src/tribler/core/components/bandwidth_accounting/tests/test_community.py

```diff
@@ -14,6 +12,10 @@
 from tribler.core.components.bandwidth_accounting.settings import BandwidthAccountingSettings
 from tribler.core.utilities.utilities import MEMORY_DB
 
+ID1 = 0
+ID2 = 1
+ID3 = 2
```

Comment:
```suggestion
ID1, ID2, ID3 = range(3)
```

This would be shorter but less explicit.

Reply:
I prefer the shorter form; it is clear enough. But both forms look good.

Reply:
Based on localized consensus (in the office), I'll adopt your short-form preference.